paddlepaddle/paddlepaddle-model-zoo/src/main/java/ai/djl/paddlepaddle/zoo/cv/wordrecognition/PpWordRecognitionTranslator.java

```diff
@@ -77,14 +77,8 @@ public Batchifier getBatchifier() {
         return null;
     }
 
-    private int[] resize32(double h, double w) {
-        double min = Math.min(h, w);
-        if (min < 32) {
-            h = 32.0 / min * h;
-            w = 32.0 / min * w;
-        }
-        int h32 = (int) h / 32;
-        int w32 = (int) w / 32;
-        return new int[] {h32 * 32, w32 * 32};
+    private int[] resize32(double w) {
+        int width = ((int) Math.max(32, w)) / 32 * 32;
```

Comment:
This is not correct. This basically keep the original width and removed the remainder.

Reply:
No, it trimmed the width to a 32 based number. The apect ratio didn't help on the inference result

Reply:
Yes